### ISSUE TITLE
MakeDb igblast is not parsing pRESTO headers

### ISSUE THREAD

**Issue by ssnn-airr** [immcantation/changeo#82]
**[Original report](https://bitbucket.org/kleinstein/changeo/issues/82) by Namita Gupta (Bitbucket: [namita1025](https://bitbucket.org/namita1025), GitHub: [namita1025](https://github.com/namita1025)).**

----------------------------------------

Both parseIMGT() and parseIgBlast() have no_parse=True by default. This is then passed to writeDb(), which uses the flag to decide whether or not to parseAnnotation from record.id.

1. Figure out where the IMGT parser is setting no_parse=False
2. Give the IgBLAST parser consistent behavior

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

This worked in v0.3.3. It looks like what happened is that at some poit this block:

```
if not no_parse:  ordered_fields.extend(list(record.annotations.keys()))
pass_handle = open(pass_file, 'wt')
pass_writer = getDbWriter(pass_handle, add_fields=ordered_fields)
```

Got moved up before the annotations were parsed, so `record.annotations` was empty.

**Comment by ssnn-airr** [comment]
**Original comment by Namita Gupta (Bitbucket: [namita1025](https://bitbucket.org/namita1025), GitHub: [namita1025](https://github.com/namita1025)).**

----------------------------------------

Ohhh this was part of Susanna making the 'partial' file so a fail handle
would exist farther up...

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

I didn't check, but that seems likely. Luckily, these means we don't need to do a release after fixing it.  Fixing it now.

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

Should be fixed in 3b8043f. Will wait to hear from @schristley before resolving.

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

Seems okay. We need to do a thorough test of MakeDb soonish.
